### PR TITLE
fix(build): create a commonJs bundle

### PR DESCRIPTION
**Pull request by spaenleh**

### CODE REVIEW

**Review by Defferrard — APPROVED** [review]
LGTM!

**Review by pyphilia — APPROVED** [review]
Thanks for the PR, I wonder if you really wanted to include some of the packages. Also I didn't know you can use comments in json?

File: package.json [hunk 1]
```diff
@@ -53,13 +65,21 @@
     "eslint": "8.57.0",
     "eslint-config-prettier": "9.1.0",
     "eslint-plugin-import": "2.29.1",
+    "fs": "0.0.1-security",
+    "glob": "10.4.2",
```
[the remark this answers is not in the document]
Comment (spaenleh) [reply package.json:69]: No, good catch !

File: tsconfig.cjs.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "outDir": "./cjs",
+    "noEmit": false,
+    // declarations are emitted separately
```
[the remark this answers is not in the document]
Comment (spaenleh) [reply tsconfig.cjs.json:6]: Yes ! It is a thing, and has been allowed in most config files (as I know of) written in JSON format (eslint, tsconfig and others). 

See this discussion from 2015 about allowing comments in tsconfig: https://github.com/microsoft/TypeScript/issues/4987

Using the wayback machine for the first link in the discussion: https://web.archive.org/web/20160316131448/https://blog.getify.com/json-comments/

**Pull request merged**